Description: REMAKE_INITRD is deprecated in dkms 3
 dkms is very noisy about it ...
Author: Andreas Beckmann <anbe@debian.org>

--- a/scripts/dkms.mkconf
+++ b/scripts/dkms.mkconf
@@ -68,7 +68,6 @@ POST_BUILD="scripts/dkms.postbuild
   -t \${dkms_tree}
 "
 AUTOINSTALL="yes"
-REMAKE_INITRD="no"
 MAKE[0]="make"
 STRIP[0]="\$(
   [[ -r \${PACKAGE_CONFIG} ]] \\
